webapp/src/app/routes/editor/block/block-checkbox/block-checkbox.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+import { BlockData, BlockValidation } from '@/core/types';
+
+@Component({
+  selector: 'block-checkbox',
+  templateUrl: './block-checkbox.component.html',
+})
+export class BlockCheckboxComponent implements OnInit {
```

Comment:
If this accepts a `validation` `@Input`, how about we call it `BlockValidationComponent`?

Reply:
All logic of the component is only about a checkbox. We even can use material checkbox, instead of the component, but it'll lead to using list of formControl, which is not nice.
Also we have other parts of our app, which go with the checkbox names, like Checkbox interface (which is not validation, because it has validation as its field).
So, my opinion is: on the stage of our app, `BlockCheckboxComponent` is better.
But later, if the component get new features (like RegExp) we can rename it to something else, e.g. `BlockValidationComponent`.